content/en/tests/explorer/saved_views.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Saved Views
+kind: documentation
```

Comment:
```suggestion
kind: documentation
aliases:
- /continuous_integration/explorer/saved_views
```
Do these need an alias back to the original location ?

Reply:
I don't believe so because we are creating net-new pages. We aren't deleting any pages, which would warrant us adding an alias to the new one.